### PR TITLE
[Storage] Use official parameters instead of aliases in examples

### LINKED ISSUE #17774: [Doc]: AccountName parameter Used in Examples but not documented
### Description

The examples to create a new storage account use the parameter `AccountName` but the documented parameter is `Name`. Either publish the undocumented Paramter `AccountName` or change the examples to use `Name`.

### PR DESCRIPTION
**Pull request by PramodValavala-MSFT**
## Description

Some of the reference docs relating to Storage Accounts like this, this, or this, use the `-AccountName` alias instead of the official `-Name` parameter. This PR addresses this for a few docs that had similar examples.

resolves #17774

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by blueww — APPROVED** [review]
The help dpc change looks good to me.
It's better we can also update [change log](https://github.com/Azure/azure-powershell/blob/main/src/Storage/Storage.Management/ChangeLog.md) for the help change.

**Comment by PramodValavala-MSFT** [comment]
@blueww Thanks! Looks like I branched out from the wrong branch. I updated the changelog and pushed.

**Review by blueww — DISMISSED** [review]

**Review by blueww — APPROVED** [review]

File: src/Storage/Storage.Management/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Updated examples in reference documentation for a few `*-AzStorageAccount*` cmdlets to use `-Name` instead of its aliases
```
Comment (isra-fel) [inline src/Storage/Storage.Management/ChangeLog.md:21]:
```suggestion
```
Document-only PR is not required to update changelog.
Online documents will be updated in a few days after official release of Azure PowerShell.

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**